src/typescript-service.ts

```diff
@@ -178,8 +178,13 @@ export class TypeScriptService {
     /**
      * Keeps track of package.jsons in the workspace
      */
-    protected packageManager: PackageManager
-
+    protected packageManager: PackageManager   
+    
+    /**
+     * The set of uris for all files currently open in the editor
+     */
+    public openFiles: Set<string>
```

Comment:
A single `typescript-service` can open multiple separate Typescript projects. The `ProjectManager` manages each project using a `ProjectConfiguration`. I would suggest you keep all project specific state inside `ProjectConfiguration`.